src/middlewared/middlewared/plugins/reporting/events.py

```diff
@@ -211,8 +211,6 @@ def run(self):
                     'time': time.monotonic(),
                     'speeds': self.get_interface_speeds(),
                 })
-            data['interfaces'] = defaultdict(dict)
```

Comment:
Wait, hoes does this fix a crash?

Reply:
@william-gr was crashing because `defaultdict` is not imported in 12.0